CHANGELOG.latest.md

```diff
@@ -1,46 +1,22 @@
+```
+Purchases.configureWith({
+    apiKey: "amazon_specific_api_key",
+    useAmazon: true,
+});
+```
 
-### Removed APIs
-- `identify` and `createAlias` have been removed in favor of `logIn`.
-- `reset` has been removed in favor of `logOut`.
-- `getEntitlements` has been removed in favor of `getOfferings`.
-- `attributionKey` and `Purchases.addAttributionData` have been removed in favor of `set<NetworkID> methods`.
-- `setAllowSharingStoreAccount` has been removed. Configure behavior through the RevenueCat dashboard instead.
+For more information around configuration please take a look at the [Amazon Appstore section in our docs](https://docs.revenuecat.com/docs/amazon-platform-resources). The official [Amazon In-App Purchasing docs](https://developer.amazon.com/docs/in-app-purchasing/iap-overview.html) also contains very valuable information, especially around testing and best practices.
 
-### Renamed APIs
-| 2.x | 3.0.0 |
-| :-: | :-: |
-| `PurchaserInfo` | `CustomerInfo` |
-| `Purchases.getPurchaserInfo` | `Purchases.getCustomerInfo` |
-| `Purchases.invalidatePurchaserInfoCache` | `Purchases.invalidateCustomerInfoCache` |
-| `PurchasesTransaction` | `PurchasesStoreTransaction` |
-| `PurchasesTransaction.revenueCatId` | `PurchasesStoreTransaction.transactionIdentifier` |
-| `PurchasesTransaction.productId` | `PurchasesStoreTransaction.productIdentifier` |
-| `PurchasesProduct` | `PurchasesStoreProduct` |
-| `PurchasesProduct.intro_price` (`number`) | `PurchasesStoreProduct.introPrice` (`PurchasesIntroPrice`) |
-| `PurchasesProduct.price_string` | `PurchasesStoreProduct.priceString` |
-| `PurchasesProduct.currency_code` | `PurchasesStoreProduct.currencyCode` |
-| `Purchases.restoreTransactions` | `Purchases.restorePurchases` |
-| `Purchases.updatedPurchaserInfoListener` | `Purchases.updatedCustomerInfoListener` |
-| `Purchases.setup` | `Purchases.configure` |
-
-### Bugfixes
-* Configure should be called in the current thread and not in a UI thread (#193) via Cesar de la Vega (@vegaro)
-### New Features
-* Add support for Firebase, Mixpanel, and CleverTap integrations (#177) via Joshua Liebowitz (@taquitos)
-* Apple AdServices support (#167)
 ### Other Changes
-* Improve purchase tester (#186) via Andy Boedo (@aboedo)
-* cordova-plugin-purchases requires Xcode 13.0+ and minimum targets iOS 11.0+ and macOS 10.13+ (#160) via NachoSoto (@NachoSoto)
-* Lots of other under-the-hood improvements. For a full overview of the changes take a look at the [full Changelog](https://github.com/revenuecat/cordova-plugin-purchases/compare/2.4.1...3.0.0)
\ No newline at end of file
+* Update orb to fix deploys (#200) via Cesar de la Vega (@vegaro)
+* Update fastlane plugin (#199) via Cesar de la Vega (@vegaro)
+* `configure` has been deprecated in favor of `configureWith`, which accepts a `PurchasesConfiguration` object (#110) via Cesar de la Vega (@vegaro)
```

Comment:
I wonder if we should mention this more prominently...

Reply:
I pushed some changes. What do you think?